### PR TITLE
minirest 0.1, curl external

### PR DESCRIPTION
**Pull request by mosteo**
Version currently in use by `alr`.

### CODE REVIEW

File: index/cu/curl/curl-external.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = "curl"
+description = "Command-line tool for transferring data with URL syntax"
+
+maintainers = ["alejandro@mosteo.com"]
+maintainers-logins = ["mosteo"]
+
+[[external]]
+kind = "system"
+origin = ["curl"]
```
Comment (Fabien-Chouteau) [inline index/cu/curl/curl-external.toml]: We can get it from the distrib

Comment (mosteo) [reply index/cu/curl/curl-external.toml]: Not sure what you mean here. That is the package in linuxes/msys2 in fact. If you mean the other external kind, this is the same situation as with `make` in which we fall back to detecting the executable when we don't have a package manager.

Comment (mosteo) [reply index/cu/curl/curl-external.toml]: Or do you mean making explicit the OS here, that's indeed probably clearer.

Comment (Fabien-Chouteau) [reply index/cu/curl/curl-external.toml]: My bad, I was confused be `orgin` is the same for all distrib.

**Pull request merged**